companion/index.ts

```diff
@@ -23,12 +23,12 @@ messaging.peerSocket.onopen = function() {
   if (val) {
     sendVal(val);
   }
-}
+};
 
 settingsStorage.onchange = function(evt) {
+  // @ts-ignore https://github.com/SergioMorchon/fitbit-sdk-types/issues/24
```

Comment:
I find this issue working wit this project :)
It's fixed in a branch, and I expect to release it with other minor fixes that I find working with random projects.